.github/configs/feature.yaml

```
@@ -1,23 +1,21 @@
 # Unless filling for special features, all features should fill for previous forks (starting from Frontier) too
 stable:
-  evm-type: stable
+  evm-type: eels
   fill-params: --until=Prague
+
 develop:
-  evm-type: develop
+  evm-type: eels
   fill-params: --until=Osaka
-static:
-  evm-type: static
+
+static: # TODO: remove and add `--fill-static-tests` to both develop & static
+  evm-type: eels
   fill-params: --until=Osaka --fill-static-tests ./tests/static
-benchmark_test:
-  evm-type: benchmark
-  fill-params: --from=Cancun --until=Prague --gas-benchmark-values 1,10,30,45,60,100,150 -m benchmark --generate-all-formats ./tests
+
+benchmark:
+  evm-type: evmone # Evmone only fully supports up to Prague
+  fill-params: --from=Cancun --until=Prague --gas-benchmark-values 1,10,30,45,60,100,150 ./tests/benchmark
```

```suggestion
  fill-params: --fork=Prague --gas-benchmark-values 1,10,30,45,60,100,150 ./tests/benchmark
```
I think we can safely remove Cancun, it takes twice as long to produce the benchmarks with it enabled and I think it serves no purpose.